.gitignore

```diff
@@ -2,6 +2,7 @@
 *.prof
 build/_output
 build/_test
+build/_tools
```

Comment:
I think we could also add `tests/_setup` to this, but we can do that in a follow-up.